ansible/cluster-firewalld.yml

```diff
@@ -0,0 +1,9 @@
+- set_fact:
```

Comment:
Minor comment - it would be useful to comment here about the intention of this playbook, ie that we want to open firewall access to all hosts on the local IP subnet.